resources/html/test/testPlayground.html

```diff
@@ -0,0 +1,188 @@
+<!--
+    This file is a html page with various element used for testing candybean's
+    functionalities
+-->
+
+<html>
+    <head>
+        <title>Candybean Test Page</title>
+        <script type="text/javascript">
+/*
+ * Toggles the visibility of the paragraph with id 'writing', can be used
+ * for testing candybean functions which require and element to variably
+ * be either visible or invisible or on/off the DOM.
+ */
+function toggleDOMWriting() {
+    var para = document.getElementById('writing');
+    var parent = document.getElementById('writingDiv');
+    if (para != null) {
+        parent.removeChild(para);
+    }
+    else{
```

Comment:
Let's make line 21 inlined with line 20, so this is consistent with our Java code in Candybean